src/docs/building-a-geospatial-app/2-scatterplot-overlay.md

```diff
@@ -1,4 +1,4 @@
-<!-- INJECT:"ScatterplotOverlay" heading -->
```

Comment:
I know this was a long time ago, but @jckr any idea of where the images are? They don't seem to be part of the repo.
See issue https://github.com/uber-common/vis-academy/issues/85 for more info.